src/settings/Settings.tsx

```diff
@@ -298,6 +299,13 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "feature_pseudonymousAnalyticsOptIn": {
```

Comment:
feature names tend to be snake_cased also this would need documenting in https://github.com/vector-im/element-web/blob/develop/docs/labs.md